### PR TITLE
feat(docs): add available integrations @ferdiunal/refine-shadcn

### PR DESCRIPTION
**Pull request by ferdiunal**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://refine.dev/docs/guides-concepts/contributing/#commit-convention

## Bugs / Features

- [ ] Related issue(s) linked
- [ ] Tests for the changes have been added
- [ ] Docs have been added / updated
- [ ] Changesets have been added https://refine.dev/docs/guides-concepts/contributing/#creating-a-changeset

### CODE REVIEW

**Comment by omeraplak** [comment]
Hey @ferdiunal @refinedev/core @refinedev/marketing ,
Thank you so much for the amazing and remarkable contribution @ferdiunal ! I think it's really important to categorize the Refine Packages correctly when listing them. How about adding them to the 'Community Packages' category?

**Comment by ferdiunal** [comment]
> Hey @ferdiunal @refinedev/core @refinedev/marketing , Thank you so much for the amazing and remarkable contribution @ferdiunal ! I think it's really important to categorize the Refine Packages correctly when listing them. How about adding them to the 'Community Packages' category?

@omeraplak I moved to community packages. Thank you for your support.

File: documentation/docs/guides-concepts/ui-libraries/index.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ These integrations are made to provide solutions for common use cases such as me
 - [Material UI with `@refinedev/mui`](/docs/ui-integrations/material-ui/introduction)
 - [Chakra UI with `@refinedev/chakra-ui`](/docs/ui-integrations/chakra-ui/introduction)
 - [Mantine with `@refinedev/mantine`](/docs/ui-integrations/mantine/introduction)
+- [Shadcn UI with `@ferdiunal/refine-shadcn`](https://github.com/ferdiunal/refine-shadcn?tab=readme-ov-file#useage)
```
Comment (BatuhanW) [inline documentation/docs/guides-concepts/ui-libraries/index.md:19]:
```suggestion
- [Shadcn UI with `@ferdiunal/refine-shadcn`](https://github.com/ferdiunal/refine-shadcn)
```

**Review by BatuhanW — APPROVED** [review]

**Review by BatuhanW — CHANGES_REQUESTED** [review]
Should be in this page: https://refine.dev/docs/packages/list-of-packages/#%EF%B8%8F-community-packages

**Review by BatuhanW — APPROVED** [review]

**Pull request merged**